### PR TITLE
improvements to rendering trees and snapshots

### PR DESCRIPTION
**Pull request by BenGardiner**
* mitigations in snapshots
* bold
* dark bg
* better support for un-numbered trees (i.e. leftover `(*)`)

### CODE REVIEW

**Comment by BenGardiner** [comment]
for example application, see https://irdetoservices.github.io/cybersecurity-for-hvedr_companion/

File: scripts/rectify-ids.py [hunk 1]
```diff
@@ -44,19 +44,7 @@ def info(type, value, tb):
...
-	node_id = node.get("id", None)
-	if not node_id is None:
-		node.update({'id': count})
-		count = count + 1
-
-	return
-
-apply_each_node(root_node, rectify_id)
-
+rectify_ids(root_node)
```
Comment (kennethblite) [inline scripts/rectify-ids.py:47]: Is this defined somewhere?

Comment (kennethblite) [reply scripts/rectify-ids.py:47]: Yep Found it.

**Comment by kennethblite** [comment]
:+1:

**Pull request merged**